internal/rbd/nodeserver.go

```diff
@@ -280,14 +283,14 @@ func (ns *NodeServer) populateRbdVol(
 
 // appendReadAffinityMapOptions appends readAffinityMapOptions to mapOptions
 // if mounter is rbdDefaultMounter and readAffinityMapOptions is not empty.
-func (ns NodeServer) appendReadAffinityMapOptions(rv *rbdVolume) {
+func appendReadAffinityMapOptions(rv *rbdVolume, readAffinityMapOptions string) {
```

Comment:
this can now be a function of the `rbdVolume` struct:
```go
func (rv *rbdVolume) appendReadAffinityMapOptions(readAffinityMapOptions string)
```
which would be cleaner